requirements/functests.in

```diff
@@ -7,3 +7,4 @@ factory-boy
 pytest-factoryboy
 h-matchers
 webtest
+filelock
```

Comment:
We're not running the functests in parallel but they share the `db_engine()` fixture which now uses `filelock`, so they need this dependency